static/js/vue-cdr-access/tests/unit/collectionBrowse.spec.js

```diff
@@ -1,15 +1,16 @@
-import { createLocalVue, shallowMount } from '@vue/test-utils'
-import VueRouter from 'vue-router';
+import { shallowMount } from '@vue/test-utils'
+import { createRouter, createWebHistory } from 'vue-router';
 import collectionBrowse from '@/components/collectionBrowse.vue';
 import moxios from "moxios";
+import displayWrapper from "@/components/displayWrapper";
 
-const localVue = createLocalVue();
-localVue.use(VueRouter);
-const router = new VueRouter({
+const router = createRouter({
+    history: createWebHistory(process.env.BASE_URL),
     routes: [
         {
-            path: '/collections/',
-            name: 'collectionBrowse'
+            path: '/record/:uuid/',
```

Comment:
Is this the right route? Previously the path was "collections" and name was "collectionBrowse"

Reply:
Hmm, this one doesn't really need a router for the tests